lib/taskana-core/src/main/java/pro/taskana/workbasket/api/WorkbasketService.java

```diff
@@ -312,18 +321,21 @@ WorkbasketAccessItem createWorkbasketAccessItem(WorkbasketAccessItem workbasketA
    *     stored ones.
    * @throws InvalidArgumentException will be thrown when the parameter {@code wbAccessItems} is
    *     NULL or member doesn't match the preconditions
-   * @throws NotAuthorizedException if the current user is not member of role {@linkplain
+   * @throws MismatchedRoleException if the current user is not member of role {@linkplain
    *     pro.taskana.common.api.TaskanaRole#ADMIN admin} or {@linkplain
    *     pro.taskana.common.api.TaskanaRole#BUSINESS_ADMIN business-admin}
```

Comment:
if the current user is not member of {@linkplain TaskanaRole#BUSINESS_ADMIN} or {@linkplain TaskanaRole#ADMIN}